hooks/command

```diff
@@ -45,7 +45,7 @@ fi
 # Parse extra env vars and add them to the docker args
 while IFS='=' read -r name _ ; do
   if [[ $name =~ ^(BUILDKITE_PLUGIN_DOCKER_ENVIRONMENT_[0-9]+) ]] ; then
-    echo "parsed env[$name] = [${!name}]"
+    echo "parsed env[$name]"
     args+=( "--env" "${!name}" )
   fi
 done < <(env | sort)
```

Comment:
I think this env parser will fail with multi-line values:

```
$ export HELLO="here's
> a
> multiline
> string"
$ env
SHELL=/bin/bash
HELLO=here's
a
multiline
string
PWD=/Users/sj26
EDITOR=vim
```

Reply:
I don't think so @sj26, the loop is just to find the environment keys, then the keys are looked up from the environment. There should have been a better comment story in there to explain that. What this will be vulnerable to is values with `BUILDKITE_PLUGIN_DOCKER_ENVIRONMENT_` in them at the start of newlines.